core/src/androidMain/kotlin/com/ustadmobile/core/shared/OnBoardingModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.ustadmobile.core.shared
```

Comment:
I think this file isn't used anymore. Is that true? If it is unused, please delete it.